libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/components/button/ButtonWithProgress.kt

```diff
@@ -42,19 +43,23 @@ import io.element.android.libraries.designsystem.theme.components.Text
 
 @Composable
 fun ButtonWithProgress(
+    text: String?,
     onClick: () -> Unit,
     modifier: Modifier = Modifier,
+    showProgress: Boolean = false,
```

Comment:
Maybe document that when `showProgress` is set to true, the lambda `onClick` will not be invoked?